opentelemetry-sdk/CHANGELOG.md

```diff
@@ -25,6 +25,10 @@
  reference or owned`LogData`. If the exporter needs to process the log data
  asynchronously, it should clone the log data to ensure it can be safely processed without
  lifetime issues.
+- Clean up public methods in SDK.
+    - [`TracerProvider::span_processors`] and [`TracerProvider::config`] was removed as it's not part of the spec.
```

Comment:
`TracerProvider::span_processors` -- is still public? Was that also intended to be pub(crate) ?

Reply:
Errr probably messed up when rebasing